src/common/options/crimson.yaml.in

```diff
@@ -121,5 +121,5 @@ options:
 - name: seastore_data_delta_based_overwrite
   type: size
   level: dev
-  desc: overwrite the existing data block based on delta if the original size is smaller than the value, otherwise do overwrite based on remapping, set to 0 to enforce the remap-based overwrite.
+  desc: overwrite the existing data block based on delta if the original size is equal to or less than the value, otherwise do overwrite based on remapping, set to 0 to enforce the remap-based overwrite.
```

Comment:
Looks "the original size" is also wrong. Is it better to be "overwrite size"?

Reply:
Done.